src/sentry/models/sentryapp.py

```diff
@@ -119,6 +119,9 @@ class SentryApp(ParanoidModel, HasApiScopes):
     date_updated = models.DateTimeField(default=timezone.now)
     date_published = models.DateTimeField(null=True, blank=True)
 
+    creator_user_id = models.IntegerField(null=True)
```

Comment:
I guess this links to `sentry.User`. We could use `FlexibleForeignKey("sentry.User", db_constraint=False)` here so that you can fetch across relations more easily.

Reply:
Actually, since it looks like you want this to hang around when users are deleted, this might not make sense.

Reply:
@wedamija Yea, exactly. I don't want this to go away. I thought doing an integer field is the best way to go.

Reply:
So if you do want the relation, one option is `on_delete=models.SET_NULL`. That'll set the column to null if the user is deleted, but keep the row. Your call though, seems like the relation isn't super important here.

Reply:
@wedamija I like that option better, let me make do that